README_maintainers.md

```diff
@@ -642,209 +624,261 @@ These commands for `-c` mean:
-8530 respectively. The agency runs on port 4001. With 2 arguments the
-first is the number of DBServers and the second is the number of 
-coordinators.
+### Dependencies
 
-If there is a third argument and it is "C", then the first coordinator
-will be started with `--console`` in a separate window (using an
-`xterm`). 
+- *Ruby*, *rspec*, *httparty* to install the required dependencies run:
+  `cd tests/rb/HttpInterface; bundler`
+- *Google Test* (compile time, shipped in the 3rdParty directory)
 
-If there is a third argument and it is "D", then all servers are started
-up in the GNU debugger in separate windows (using `xterm`s). In that
-case one has to hit ENTER in the original terminal where the script runs
-to continue, once all processes have been start up in the debugger.
+### Folder Locations
 
-ArangoDB on Mesos
-=================
+There are several major places where unittests live:
 
-This will spawn a **temporary** local mesos cluster.
+| Path                         | Description
+|:-----------------------------|:-----------------------------
+| `tests/js/server/`           | JavaScript tests, runnable on the server
+| `tests/js/common/`           | JavaScript tests, runnable on the server & via arangosh
+| `tests/js/common/test-data/` | Mock data used for the JavaScript tests
+| `tests/js/client/`           | JavaScript tests, runnable via arangosh
+| `tests/rb/`                  | rspec tests (Ruby)
+| `tests/rb/HttpInterface/`    | rspec tests using the plain RESTful interface of ArangoDB. Include invalid HTTP requests and error handling checks for the server.
+| `tests/` (remaining)         | Google Test unittests
 
-Requirements:
+### Filename conventions
 
-- Somewhat recent linux
-- docker 1.10+
-- curl
-- sed (for file editing)
-- jq (for json parsing)
-- git
-- at least 8GB RAM
-- fully open firewall inside the docker network
+Special patterns in the test filenames are used to select tests to be executed
+or skipped depending on parameters:
 
-To startup a local mesos cluster:
+| Substring       | Description
+|:----------------|:----------------
+| `-server`       | Make use of existing external server. Example: `scripts/unittest http_server --server tcp://127.0.0.1:8529/`
+| `-cluster`      | These tests will only run if clustering is tested (option 'cluster' needs to be true).
+| `-noncluster`   | These tests will only run if no cluster is used (option 'cluster' needs to be false)
+| `-timecritical` | These tests are critical to execution time - and thus may fail if arangod is to slow. This may happen i.e. if you run the tests in valgrind, so you want to avoid them since they will fail anyways. To skip them, set the option `skipTimeCritical` to *true*.
+| `-disabled`     | These tests are disabled. You may however want to run them by hand.
+| `replication`   | These tests aren't run automatically since they require a manual set up environment (only applies to ruby tests).
```

Comment:
remove the replication column. its obsolete.

Reply:
removed the row